bugwarrior/services/gitlab.py

```diff
@@ -433,7 +434,19 @@ def include_todo(todo):
 
     def issues(self):
         tmpl = '{scheme}://{host}/api/v4/projects'
-        all_repos = self._fetch_paged(tmpl)
+
+        all_repos = []
+        if self.include_repos:
+            for repo in self.include_repos:
+                indiv_tmpl = tmpl + '/' + quote_plus(repo)
+                item = self._fetch(indiv_tmpl)
+                if not item:
+                    break
+
+                all_repos += [ item ]
+        else:
+            all_repos = self._fetch_paged(tmpl)
+
         repos = list(filter(self.filter_repos, all_repos))
```

Comment:
`include_repos` is used in the `filter_repos` function. You should remove this line here and rename `all_repos` to `repos`.

Edit: ... otherwise, you're filtering twice.

Reply:
... or maybe even leave the code here as is and remove the `include_repos` filtering from `filter_repos`, because `filter_repos` also filters the `exclude_repos`.

Reply:
What about renaming the `all_repos` on 446 to repos and moving the `self.filter_repos` bit up in to the else? That way, I'm only filtering if it went through the normal fetch methods?

Reply:
Then, you would leave out the `exclude_repos` and the `exclude_regex` filters. All of these four configuration settings can be used at the same time, e.g.:

```ini
gitlab.include_regex = foo/bar/*
gitlab.exclude_repos = foo/bar/baz
```

Get everything from the **foo/bar** sub-group except this one **baz** project.

Reply:
Removed include_repos filter from filter_repos check in 3417384